Update BigQuery OAuth docs with Redirect URI UI (#5698)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-bigquery-oauth.md

To set up BigQuery OAuth in dbt Cloud, a BigQuery admin must:

> 1. [Reference the connection's OAuth 2.0 Settings](#reference-the-connections-oauth-settings-in-dbt-cloud) in dbt Cloud.

Reviewer: ```suggestion
1. [Locate the redirect URI setting](#locate-the-redirect-uri-setting) in dbt Cloud.
```

2. [Create a BigQuery OAuth 2.0 client ID and secret](#creating-a-bigquery-oauth-20-client-id-and-secret) in BigQuery.
3. [Configure the connection](#configure-the-connection-in-dbt-cloud) in dbt Cloud.

To use BigQuery in the dbt Cloud IDE, all developers must:
1. [Authenticate to BigQuery](#authenticating-to-bigquery) in the their profile credentials.

> ### Reference the connection's OAuth 2.0 Settings

Reviewer: sugg using a more descriptive section heading (less generic) like this or something similar -- up to you

```suggestion
### Locate the redirect URI setting
```

> To get started, locate the connection's Redirect URI for configuring BigQuery OAuth. To do so:

Reviewer: sugg using lower case in this instance since it's not a proper noun

```suggestion
To get started, locate the connection's redirect URI for configuring BigQuery OAuth. To do so:
```

>  - go to Settings by clicking the gear in the top right.

Reviewer: sugg updating to latest UI

```suggestion
 - Select the gear menu in the upper left corner and choose **Account settings**
```

>  - on the left, select **Projects** under **Account Settings**

Reviewer: ```suggestion
 - From the left sidebar, select **Projects** 
```

>  - choose your project from the list

Reviewer: ```suggestion
 - Choose your project from the list
```

>  - select **Connection** to edit the connection details

Reviewer: ```suggestion
 - Select **Connection** to edit the connection details
```

>  - locate **Redirect URI** under the **OAuth 2.0 Settings** section

Reviewer: ```suggestion
 - Locate the **Redirect URI** option under the **OAuth 2.0 Settings** section. Copy this value to your clipboard to use later on. 
```

Set up an [OAuth consent screen](https://support.google.com/cloud/answer/6158849) if you haven't already. Then, click **+ Create Credentials** at the top of the page and select **OAuth client ID**.

> Fill in the client ID configuration. **Authorized JavaScript Origins** are not applicable. Add an item to **Authorized redirect URIs**, replacing `REDIRECT_URI` with value copied from the connection's **OAuth 2.0 Settings** section in dbt Cloud:

Reviewer: ```suggestion
Fill in the client ID configuration. **Authorized JavaScript Origins** are not applicable. Add an item to **Authorized redirect URIs** and replace `REDIRECT_URI` with the value you copied to your clipboard earlier from the connection's **OAuth 2.0 Settings** section in dbt Cloud:
```

| Config | Value |
| ------ | ----- |
| **Application type** | Web application |
| **Name** | dbt Cloud |
| **Authorized Javascript origins** | https://YOUR_ACCESS_URL |
| **Authorized Redirect URIs** | https://YOUR_ACCESS_URL/complete/bigquery |
Fill in the client ID configuration. **Authorized JavaScript Origins** are not applicable. Add an item to **Authorized redirect URIs**, replacing `REDIRECT_URI` with value copied from the connection's **OAuth 2.0 Settings** section in dbt Cloud:

| Config                       | Value           |
| ---------------------------- | --------------- |
| **Application type**         | Web application |
| **Name**                     | dbt Cloud       |
| **Authorized redirect URIs** | `REDIRECT_URI`  |

Then click **Create** to create the BigQuery OAuth app and see the app client ID and secret values. These values are available even if you close the app screen, so this isn't the only chance you have to save them.

> <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/dbt-cloud-enterprise/BQ-auth/bq-oauth-app.gif" title="Creating an OAuth app in BigQuery" />

Reviewer: should this GIF be removed? if this GIF is not being used elsewhere, we can delete it to keep things tidy/clean in our repo

Author: yep, I'll delete old images

<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/dbt-cloud-enterprise/BQ-auth/bq-oauth-app-02.png" title="Create an OAuth app in BigQuery" />

### Configure the Connection in dbt Cloud
Now that you have an OAuth app set up in BigQuery, you'll need to add the client ID and secret to dbt Cloud. To do so:
 - go to Settings by clicking the gear in the top right.
 - on the left, select **Projects** under **Account Settings**
 - choose your project from the list
 - select **Connection** to edit the connection details
 - add the client ID and secret from the BigQuery OAuth app under the **OAuth2.0 Settings** section

> <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/dbt-cloud-enterprise/BQ-auth/dbt-cloud-bq-id-secret.gif" title="Adding BigQuery OAuth application client ID and secret to dbt Cloud" />

Reviewer: same thing here. can this GIF be deleted?

>  - return to the connection details page as described in [referencing the connection's OAuth URIs](#reference-the-connections-oauth-uris)

Reviewer: ```suggestion
 - Navigate back to the connection details page, as described in [Locate the redirect URI setting](#locate-the-redirect-uri-setting)
```

>  - add the client ID and secret from the BigQuery OAuth app under the **OAuth 2.0 Settings** section

Reviewer: ```suggestion
 - Add the client ID and secret from the BigQuery OAuth app under the **OAuth 2.0 Settings** section
```
